.github/workflows/compilation.yml

```diff
@@ -56,23 +56,8 @@ jobs:
         cd build-test
         ctest --verbose --output-on-failure
 
-    - name: Restore omf2097-assets.zip
-      id: cache-assets
-      uses: actions/cache/restore@v4
-      with:
-        path: omf2097-assets.zip
-        key: omf2097-assets.zip
-
-    - name: Download omf 2097 assets
-      if: steps.cache-assets.outputs.cache-hit != 'true'
-      run: wget -q "${{ env.OPENOMF_ASSETS_URL }}"
-
-    - name: Cache omf2097-assets.zip
-      if: steps.cache-assets.outputs.cache-hit != 'true'
-      uses: actions/cache/save@v4
-      with:
-        path: omf2097-assets.zip
-        key: omf2097-assets.zip
+    - name: Get OMF2097 assets
+      uses: ./.github/actions/assets
```

Comment:
Is the `./` part in the beginning needed? Seems maybe it isn't? So I'd prefer just `.github/actions/assets` if it works.

Reply:
I believe I had to do it when I did the version `uses`